core/ui/TestCases/DefaultTemplate.tid

```diff
@@ -0,0 +1,22 @@
+title: $:/core/ui/testcases/DefaultTemplate
+
+\whitespace trim
+<$let
+	state={{{ [<qualify "$:/state/testcase">] }}}
+>
+	<div class="tc-testcase-wrapper">
+		<div class="tc-testcase-header">
+			<h2><$transclude tiddler="Description" mode="inline"/></h2>
+		</div>
+		<div class="tc-testcase-panes">
+			<div class="tc-testcase-source">
+				<$macrocall $name="tabs" tabsList="[all[tiddlers]sort[]] -[prefix<state>] -Description -ExpectedResult -Output Output +[putfirst[]] -[has[plugin-type]]" state=<<state>> default="Output" template="$:/core/ui/testcases/DefaultTemplate/Source"/>
+			</div>
+			<div class="tc-testcase-divider">
+			</div>
+			<div class="tc-testcase-output">
```

Comment:
It be useful to add a linkcatcher widget here and select the tab corresponding to the clicked link?

Reply:
Thank you @btheado I've belatedly implemented your suggestion in Thank you @btheado in 648855e8a50b1ee3bef3120b64b5713b69190c9b

Reply:
Thanks. Works great!